AUTHORS.txt

```diff
@@ -2,19 +2,23 @@ Adrien Devresse
 Alexandru Săvulescu
 Ali Can Demiralp
 Angelos Plastropoulos
+Chris Byrohl
+Chris De Grendele
 contre
```

Comment:
Maybe not now, but if we just have Github usernames, should we credit those with the `@` in front?